modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc

```diff
@@ -282,26 +288,62 @@ This instance is NOT ready to be migrated to Kiteworks!
 
 === Migration
 
-After all prerequisites, installations, configurations and the verification has passed, you can initiate the migration process. This process may take some time to copy all files to the new system. Note that a migration run can be interrupted harmlessly at any time. Starting a new migration run will continue where the previous one stopped. For a possible improvement of transfer performance, read the xref:tuning-transfer-performance[Tuning Transfer Performance] section below. Issue the following command to start the migration:
+After all prerequisites, installations, configurations and the verification has passed, you can initiate the migration process. The migration is split into four parts which are:
+
+[,subs="+callouts,macros,attributes+"]
+----
+1. Migrate users <1>
+2. Migrate files <1>
+3. Migrate shares <2>
+4. Disable users <3>
+----
+<1> These steps migrate all ordinary, guest and disabled users and files. You can rerun these steps to migrate any items that were not part when the respective step was called. Note that the steps must be made in that order.
```

Comment:
```suggestion
<1> These steps migrate all ordinary, guest and disabled users and files. You can rerun these steps to migrate any items that did not exist when the respective step was called. Note that the steps must be made in that order.
```

Or maybe "that did not exist, or failed to migrate when..."

Reply:
I have updated you suggestion to `that did not exist, or failed to migrate` so I can commit it.